WooCommerce/Classes/Authentication/Store Creation/Free Trial/FreeTrialSummaryView.swift

```diff
@@ -0,0 +1,182 @@
+        .init(title: Localization.Feature.support, icon: .supportIcon),
+        .init(title: Localization.Feature.backups, icon: .backupsIcon),
+        .init(title: Localization.Feature.security, icon: .cloudOutlineImage),
+        .init(title: Localization.Feature.launch, icon: .customizeDomainsImage)
+    ]
+
+    /// Closure invoked when the close button is pressed
+    ///
+    let onClose: (() -> ())?
+
+    /// Closure invoked when the "Try For Free"  button is pressed
+    ///
+    let onContinue: (() -> ())?
+
+    var body: some View {
+        VStack(spacing: .zero) {
+            // Main Content
+            ScrollView {
+                VStack(alignment: .leading, spacing: .zero) {
+
+                    // Illustration header & Close Button
+                    HStack(alignment: .top) {
+                        Button {
+                            onClose?()
+                        } label: {
+                            Image(uiImage: .closeButton)
+                                .foregroundColor(Color(.textSubtle))
+                        }
+
+                        Spacer()
+
+                        Image(uiImage: .freeTrialIllustration)
+                    }
+                    .padding([.trailing, .bottom], Layout.illustrationInset)
+
+                    // Title
+                    Text(Localization.launchInDays)
+                        .bold()
+                        .titleStyle()
+                        .padding(.bottom, Layout.titleSpacing)
+                        .padding(.trailing, Layout.estimatedIllustrationWidth)
+
+                    // Description
+                    Text(Localization.weOfferEverything)
+                        .secondaryBodyStyle()
+                        .padding(.trailing, Layout.infoTrailingMargin)
+                        .padding(.bottom, Layout.sectionsSpacing)
+
+                    // Features
+                    Text(Localization.tryItForDays)
+                        .bold()
+                        .secondaryTitleStyle()
+                        .padding(.bottom, Layout.titleSpacing)
+
+                    ForEach(Self.features, id: \.title) { feature in
+                        HStack {
+                            Image(uiImage: feature.icon)
+                                .foregroundColor(Color(uiColor: .accent))
+
+                            Text(feature.title)
+                                .foregroundColor(Color(.text))
+                                .calloutStyle()
+                        }
+                        .padding(.bottom, Layout.featureSpacing)
+                    }
+
+                    // WPCom logo
+                    HStack {
+                        Text(Localization.poweredBy)
+                            .foregroundColor(Color(uiColor: .textSubtle))
+                            .captionStyle()
+
+                        Image(uiImage: .wpcomLogoImage(tintColor: .textSubtle))
+
+                    }
```

Comment:
This is challenging for localization, because not all languages use this word order (with `WordPress.com` at the end of the phrase). I'm not sure if there's a way we could insert just the logo into a string, but maybe it would be ok to place just the logo (without text) at the beginning or end of a complete string ("Powered by WordPress.com")?